ibm/mas_devops/roles/suite_app_install/vars/defaultspecs/predict.yml

```diff
@@ -1,5 +1,10 @@
 ---
-# Default application spec for Predict
+# Default application specs for Predict
 mas_app_spec:
   bindings:
+    health: workspace
```

Comment:
health and monitor are not valid bindings.  

1) If the application actually needs this information it needs to be fixed
2) This should be validated as an invalid CRD because these are not valid things to bind to